egret/model_library/unit_commitment/uc_model_generator.py

```diff
@@ -88,7 +88,7 @@ def _generate_model( model_data,
     model.name = "UnitCommitment"
     
     ## munge ptdf_options, if necessary
-    if _power_balance in ['lazy_ptdf_power_flow', 'ptdf_power_flow']:
+    if _power_balance in ['ptdf_power_flow']:
```

Comment:
So the full-bore, up-front PTDF power flow is no longer an option? I agree that one probably doesn't want to mess with this in practice, but it might be useful to retain for purposes of a comparative baseline.

Reply:
It is, they're just handled in the same power_balance function now by specifying `'lazy'=False` in `ptdf_options`.

Reply:
Then we're good. Thanks!